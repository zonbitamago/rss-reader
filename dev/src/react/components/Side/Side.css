.Side {
  background-color: #01579b;
  width: 3em;
  height: 100vh;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
}
