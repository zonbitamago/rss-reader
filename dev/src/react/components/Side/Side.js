import React, { Component } from "react";
import PropTypes from "prop-types";
import styles from "./Side.css";
import Menus from "../Menus/Menus";
import Time from "../Time/Time";

class Side extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="Side">
        <div className="Side-Top">
          <Menus electronUtil={this.props.electronUtil} />
        </div>
        <div className="Side-Bottom">
          <Time />
        </div>
      </div>
    );
  }
}

Side.propTypes = { electronUtil: PropTypes.object };

Side.defaultProps = {};

export default Side;
