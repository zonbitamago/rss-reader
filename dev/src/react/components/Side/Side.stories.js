import React from "react";
import { storiesOf } from "@storybook/react";

import Side from "./Side";

storiesOf("Organisms/Side", module).add("Side", () => <Side />);
