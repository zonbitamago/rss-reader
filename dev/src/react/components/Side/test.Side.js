import React from 'react'
import {
  shallow
} from 'enzyme'
import {
  electronUtil
} from "../../../../__mocks__/electronUtilSettings";

import Side from './Side'

describe('Side', () => {
      let component, props

      beforeEach(() => {
          props = {
            electronUtil: electronUtil
          }
          component = shallow( < Side { ...props
            }
            />)
          })

        it('should', () => {
          expect(component).toMatchSnapshot()
        })
      })