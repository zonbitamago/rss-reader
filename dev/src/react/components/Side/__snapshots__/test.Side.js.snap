// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Side should 1`] = `
<div
  className="Side"
>
  <div
    className="Side-Top"
  >
    <Menus />
  </div>
  <div
    className="Side-Bottom"
  >
    <Time />
  </div>
</div>
`;
